apps/openmw/mwworld/projectilemanager.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <string>
 
+#include <osg/observer_ptr>
```

Comment:
We don't. This could be moved to ` apps/openmw/mwworld/projectilemanager.cpp`.

Here is the error I get when it is missing from either file:
```
In file included from ../apps/openmw/mwworld/projectilemanager.cpp:1:
In file included from ../apps/openmw/mwworld/projectilemanager.hpp:4:
In file included from /usr/include/c++/v1/string:439:
In file included from /usr/include/c++/v1/algorithm:627:
/usr/include/c++/v1/utility:253:9: error: implicit instantiation of undefined template 'osg::observer_ptr<osg::Camera>'
    _T1 first;
        ^
/usr/include/c++/v1/map:629:16: note: in instantiation of template class 'std::__1::pair<const osg::observer_ptr<osg::Camera>, std::__1::vector<SceneUtil::LightManager::LightSourceViewBound, std::__1::allocator<SceneUtil::LightManager::LightSourceViewBound> > >' requested here
    value_type __cc;
               ^
/usr/include/c++/v1/__tree:1440:14: note: in instantiation of template class 'std::__1::__value_type<osg::observer_ptr<osg::Camera>, std::__1::vector<SceneUtil::LightManager::LightSourceViewBound, std::__1::allocator<SceneUtil::LightManager::LightSourceViewBound> > >' requested here
    if (__nd != nullptr)
             ^
/usr/include/c++/v1/__tree:1433:5: note: in instantiation of member function 'std::__1::__tree<std::__1::__value_type<osg::observer_ptr<osg::Camera>, std::__1::vector<SceneUtil::LightManager::LightSourceViewBound, std::__1::allocator<SceneUtil::LightManager::LightSourceViewBound> > >, std::__1::__map_value_compare<osg::observer_ptr<osg::Camera>, std::__1::__value_type<osg::observer_ptr<osg::Camera>, std::__1::vector<SceneUtil::LightManager::LightSourceViewBound, std::__1::allocator<SceneUtil::LightManager::LightSourceViewBound> > >, std::__1::less<osg::observer_ptr<osg::Camera> >, true>, std::__1::allocator<std::__1::__value_type<osg::observer_ptr<osg::Camera>, std::__1::vector<SceneUtil::LightManager::LightSourceViewBound, std::__1::allocator<SceneUtil::LightManager::LightSourceViewBound> > > > >::destroy' requested here
    destroy(__root());
    ^
/usr/include/c++/v1/__tree:1119:78: note: in instantiation of member function 'std::__1::__tree<std::__1::__value_type<osg::observer_ptr<osg::Camera>, std::__1::vector<SceneUtil::LightManager::LightSourceViewBound, std::__1::allocator<SceneUtil::LightManager::LightSourceViewBound> > >, std::__1::__map_value_compare<osg::observer_ptr<osg::Camera>, std::__1::__value_type<osg::observer_ptr<osg::Camera>, std::__1::vector<SceneUtil::LightManager::LightSourceViewBound, std::__1::allocator<SceneUtil::LightManager::LightSourceViewBound> > >, std::__1::less<osg::observer_ptr<osg::Camera> >, true>, std::__1::allocator<std::__1::__value_type<osg::observer_ptr<osg::Camera>, std::__1::vector<SceneUtil::LightManager::LightSourceViewBound, std::__1::allocator<SceneUtil::LightManager::LightSourceViewBound> > > > >::~__tree' requested here
    template <class, class, class, class> friend class _LIBCPP_TYPE_VIS_ONLY map;
                                                                             ^
/usr/local/include/osg/ref_ptr:21:28: note: template is declared here
template<typename T> class observer_ptr;
                           ^
```